### ISSUE TITLE
Failed to enable "Board Statistics" with Extensions Manager in phpBB-3.1.10

### ISSUE THREAD

**Issue by zhengfish** [ForumHulp/upload#1]
There is some error as below:

Information
Forumhulp helper application does not exist!
Download forumhulp/helper and copy the helper folder to your forumhulp extension folder.

**Comment by ForumHulp** [comment]
Do as it is asking you and install https://github.com/ForumHulp/helper in your ext folder.

 

Greetz

 

 

Van: zhengfish [mailto:notifications@github.com] 
Verzonden: woensdag 8 februari 2017 07:35
Aan: ForumHulp/upload <upload@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: [ForumHulp/upload] Failed to enable "Board Statistics" with Extensions Manager in phpBB-3.1.10 (#1)

 

There is some error as below:

Information
Forumhulp helper application does not exist!
Download forumhulp/helper and copy the helper folder to your forumhulp extension folder.

**Comment by zhengfish** [comment]
Hello,
   Yes, you are right.
   I fixed this issue after I download and copy this helper to /ext/helper.

   However I still cannot get this plugin run.
   My box is located in company network and cannot access internet directly.
   I found this error message:
   [phpBB Debug] PHP Warning: in file
[ROOT]/ext/forumhulp/statistics/event/listener.php on line 183:
fsockopen(): php_network_getaddresses: getaddrinfo failed: Name or service
not known
   [phpBB Debug] PHP Warning: in file
[ROOT]/ext/forumhulp/statistics/event/listener.php on line 183:
fsockopen(): unable to connect to www.ip-api.com:80
(php_network_getaddresses: getaddrinfo failed: Name or service not known)

   This plugin need to access internet before it works?

Thanks in advance.
